### DISCUSSION TITLE
ValueError: `decoder_start_token_id` or `bos_token_id` has to be defined for encoder-decoder generation

### DISCUSSION THREAD

**Discussion by xcc2731594** [amazon-science/chronos-forecasting#170] [General]
ValueError: `decoder_start_token_id` or `bos_token_id` has to be defined for encoder-decoder generation

**Comment by lostella** [comment]
@xcc2731594 did you try updating `transformers`? It introduced this issue in the past (see [here](https://github.com/huggingface/transformers/issues/30892)) which seems related.

Also, consider opening an [issue](https://github.com/amazon-science/chronos-forecasting/issues) for anything not working correctly in the code, instead of a discussion.
